centralized-sampling-tests/sample-apps/golang-http-server/application.go

```diff
@@ -0,0 +1,160 @@
+		userAttribute := r.Header.Get("User")
+		required := r.Header.Get("Required")
+		serviceName := r.Header.Get("Service_name")
+		totalSpans := r.Header.Get("Totalspans")
+
+		var attributes = []attribute.KeyValue{
+			attribute.KeyValue{"http.method", attribute.StringValue(r.Method)},
+			attribute.KeyValue{"http.url", attribute.StringValue("http://localhost:8080/getSampled")},
+			attribute.KeyValue{"user", attribute.StringValue(userAttribute)},
+			attribute.KeyValue{"http.route", attribute.StringValue("/getSampled")},
+			attribute.KeyValue{"required", attribute.StringValue(required)},
+			attribute.KeyValue{"http.target", attribute.StringValue("/getSampled")},
+		}
+
+		var totalSampled = getSampledSpanCount(serviceName, totalSpans, attributes)
+		_, err := w.Write([]byte(strconv.Itoa(totalSampled)))
+		if err != nil {
+			log.Println(err)
+		}
+	}))
+
+	http.Handle("/importantEndpoint", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		userAttribute := r.Header.Get("User")
+		required := r.Header.Get("Required")
+		serviceName := r.Header.Get("Service_name")
+		totalSpans := r.Header.Get("Totalspans")
+
+		var attributes = []attribute.KeyValue{
+			attribute.KeyValue{"http.method", attribute.StringValue("GET")},
+			attribute.KeyValue{"http.url", attribute.StringValue("http://localhost:8080/importantEndpoint")},
+			attribute.KeyValue{"user", attribute.StringValue(userAttribute)},
+			attribute.KeyValue{"http.route", attribute.StringValue("/importantEndpoint")},
+			attribute.KeyValue{"required", attribute.StringValue(required)},
+			attribute.KeyValue{"http.target", attribute.StringValue("/importantEndpoint")},
+		}
+
+		var totalSampled = getSampledSpanCount(serviceName, totalSpans, attributes)
+		_, err := w.Write([]byte(strconv.Itoa(totalSampled)))
+		if err != nil {
+			log.Println(err)
+		}
+	}))
+
+	listenAddress := os.Getenv("LISTEN_ADDRESS")
+	if listenAddress == "" {
+		listenAddress = "localhost:8080"
+	}
+	log.Println("App is listening on %s !", listenAddress)
+	_ = http.ListenAndServe(listenAddress, nil)
+}
+
+func start_xray() (bool, error) {
+	ctx := context.Background()
+
+	exporterEndpoint := os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")
+	if exporterEndpoint == "" {
+		exporterEndpoint = "localhost:4317"
+	}
+
+	log.Println("Creating new OTLP trace exporter...")
+	traceExporter, err := otlptracegrpc.New(ctx, otlptracegrpc.WithInsecure(), otlptracegrpc.WithEndpoint(exporterEndpoint), otlptracegrpc.WithDialOption(grpc.WithBlock()))
+	if err != nil {
+		log.Fatalf("Failed to create new OTLP trace exporter: %v", err)
+		return false, err
+	}
+
+	idg := xray.NewIDGenerator()
+
+	samplerEndpoint := os.Getenv("XRAY_ENDPOINT")
+	if samplerEndpoint == "" {
+		samplerEndpoint = "http://localhost:2000"
+	}
+	endpointUrl, err := url.Parse(samplerEndpoint)
+
+	res, err := sampler.NewRemoteSampler(ctx, "aws-otel-integ-test", "", sampler.WithEndpoint(*endpointUrl), sampler.WithSamplingRulesPollingInterval(10*time.Second))
```

Comment:
```suggestion
	res, err := sampler.NewRemoteSampler(ctx, "adot-integ-test", "", sampler.WithEndpoint(*endpointUrl), sampler.WithSamplingRulesPollingInterval(10*time.Second))
```
nit: Sorry I didn't notice this the first time. Let's try to use `adot` instead of `aws-otel` wherever we can.